recipe/meta.yaml

```diff
@@ -36,7 +38,7 @@ test:
     - psycopg2._psycopg
 
 about:
-  home: http://initd.org/psycopg/
+  home: https://www.psycopg.org/
   license: LGPL, BSD-like, ZPL
```

Comment:
I guess the license should be `LGPL-3.0+ AND Other` because a BSD-like license is not quite clear one

Reply:
I think the way to declare `LGPL-3.0+` is ` LGPL-3.0-or-later`.

See here for what the `anaconda-linter` will be using: https://spdx.org/licenses/